src/screens/transaction-details/TransactionDetails.tsx

```diff
@@ -24,14 +26,25 @@ type Params = {
 
 export const TransactionDetails: React.FC = () => {
   const route = useRoute();
+  const { colors } = useTheme();
   const { transaction } = route.params as Params;
   const { height } = useDimensions();
 
+  const hash = ethereumUtils.getHash(transaction);
+  const fee = transaction.fee;
+
   return (
-    <Container height={TRANSACTION_DETAILS_SHEET_HEIGHT} deviceHeight={height}>
+    <Container
```

Comment:
why do you need the container here?

Reply:
I think the sheet was acting weird when there was no container. I would appreciate any tips related to using sheets. For now I just set it up so that I can implement the separate components and was planning to figure out proper height calculations at the end when the whole UI is implemented.